daprdocs/content/en/js-sdk-docs/js-client/_index.md

```diff
@@ -37,11 +39,41 @@ const server = new DaprServer(serverHost, serverPort, daprHost, daprPort, Commun
 const client = new DaprClient(daprHost, daprPort, CommunicationProtocolEnum.GRPC);
 ```
 
-##### DaprClient Library
+## Running using HTTP or gRPC
```

Comment:
I thought after a discussion we had we could use something like this? What are your thoughts? @amulyavarote 

## Running 

To run the examples, you can use 2 different protocols to interact with the Dapr sidecar. This can either be HTTP (default) or gRPC.

### Using HTTP (default)

```javascript
import { DaprClient, DaprServer } from "dapr-client";
const client = new DaprClient(daprHost, daprPort);
const server= new DaprServer(appHost, appPort, daprHost, daprPort);
```

```bash
# Using dapr run
dapr run --app-id <example-sdk> --app-port 50051 --app-protocol http npm run start
# or, using npm script
npm run start:dapr-http
```

### Using gRPC

As for gRPC. since HTTP is the default, we will have to adapt the communication protocol, which we can do by passing an extra argument to the client or server constructor.

```javascript
import { DaprClient, DaprServer, CommunicationProtocol } from "dapr-client";
const client = new DaprClient(daprHost, daprPort, CommunicationProtocol.GRPC);
const server= new DaprServer(appHost, appPort, daprHost, daprPort, CommunicationProtocol.GRPC);
```

```bash
  # Using dapr run
  dapr run --app-id <example-sdk> --app-port 50051 --app-protocol grpc npm run start
  # or, using npm script
  npm run start:dapr-grpc
```

Reply:
Agreed! Thanks @XavierGeerinck